src/content/reference/react/useMemo.md

```diff
@@ -34,29 +34,29 @@ function TodoList({ todos, tab }) {
 }
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh lainnya.](#usage)
```

Comment:
```suggestion
[Lihat contoh lainnya di bawah ini.](#usage)
```

For Consistency with:
https://id.react.dev/reference/react/useContext#reference
https://id.react.dev/reference/react/useTransition#reference